### PR TITLE
Restaurant page

### PR DESCRIPTION
**Pull request by billodiallo**
In this project I created a simple restaurant page using JS6 and finished the following milestones:

Set up an HTML skeleton inside of dist/index.html with a single div element.
Set the style, image file loaders with Webpack.
Create homepage for a restaurant with a homepage, menu page and contact page.
Each module exports a function that creates an element, adds the appropriate content and styles to that element and then appends it to the DOM.
Navbar is created and
Tab-switching logic for navbar with the listener is created.
Implement PubSup pattern to populate specific pages on a specific time.

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
+import './style.css';
+import headerPage from './header';
+import renderHome from './home';
+import foodpage from './food';
+import contactPage from './contact';
+
+// console.log('hello word');
+
+function fordbd() {
```
Comment (rindrajosia) [inline src/index.js]: Use arrow functions instead of the function keyword

File: src/index.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+import './style.css';
+import headerPage from './header';
+import renderHome from './home';
+import foodpage from './food';
+import contactPage from './contact';
+
+// console.log('hello word');
```
Comment (rindrajosia) [inline src/index.js]: Comments: It’s better to delete commented piece of codes.

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- Good UI :+1: 

To improve :factory_worker: [REQUIRED]:

- [x] Please, make sure that live demo link and MIT link are not broken. Keep in mind that the first thing user checks is the readme file so, lets try to make it professional :smiley: 
- [x] It will be nice if you use this file [.github/workflows/linters.yml ](https://github.com/microverseinc/linters-config/blob/master/javascript/.github/workflows/linters.yml)  in your project.
- [x] Inline comments: It’s better to delete commented code. If you need to review a previous version of your code, use git history.(eg: // console.log('hello word');)
- [x] Please use `arrow functions` instead of the `function keyword` as you are using ES6.  ([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/javascript/restaurant_page/es6_requirement.png) from [Pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/59622983/)).

Nice to have:

- [ ] 

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

**Comment by billodiallo** [comment]
Hello,
change done.
thanks for the review.
BR//

**Review by MuriloRoque — APPROVED** [review]
## Status: Approved :green_circle: 

### Hello @billodiallo :wave: 

### Thank you for making all of the requested changes. Your project is approved :clap: :clap: :clap: 

### You may merge, good luck with your next projects :rocket: 

Murilo Roque Paiva da Silva

- Github: [@muriloroque](https://github.com/MuriloRoque)
- Linkedin: [MuriloRoque](https://www.linkedin.com/in/murilo-roque-b1268741/)

[Twitter Follow](https://twitter.com/MuriloRoquePai1)

**Pull request merged**